MBBSEmu/HostProcess/Enums/UserStatus.cs

```diff
@@ -0,0 +1,37 @@
+namespace MBBSEmu.HostProcess.Enums
+{
+    /// <summary>
+    /// User Status values, mostly taken from BRKTHU.H
+    /// </summary>
+    public enum UserStatus
```

Comment:
Rename to `EnumUserStatus` to keep consistent Enum naming convention

Reply:
I thought Polish notation was dead?